incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -470,81 +490,89 @@ would require them to set the `allowCascadingDeletion` property of the child
 namespace.
 
 <a name="admin-types"/>
+<a name="admin-resources"/>
 
-### Modify the object types propagated by HNC
+### Modify the resources propagated by HNC
 
-Starting from HNC v0.6, HNC supports the following propagation modes for each
-resource:
-* `Propagate`: propagates objects from ancestors to descendants and deletes
-  obsolete descendants.
-* `Remove`: deletes all existing propagated copies.
-* `Ignore`: stops modifying this resource. New or changed objects will not be
-  propagated, and obsolete objects will not be deleted. The `inherited-from`
-  label is not removed. Any unknown mode is treated as `Ignore`.
+HNC is configured via the [`HNCConfiguration`](#admin-access) object.  You can
+inspect this object directly via `kubectl get -oyaml hncconfiguration config`,
+or with the HNS plugin via `kubectl hns config describe`.
+
+The most important type of configuration is the way each object type
+("resource") is synchronized across namespace hierarchies. This is known as the
+"synchronization mode," and has the following options:
+
+* **Propagate:** propagates objects from ancestors to descendants and deletes
+  obsolete descendants. This is the default if a resource is listed in the
+  config but no mode is explicitly set.
```

Comment:
Since we have schema validation there's no way to have empty mode. Maybe remove `the default` part?

Reply:
Didn't realize that - thanks! Done.